content/java/concepts/array-list/terms/length/length.md

```diff
@@ -0,0 +1,40 @@
+---
+Title: '.length()'
+Description: 'Returns the length of a string, in terms of number of characters.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'String'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+In Java, the `.length()` method is a member of the `String` class that returns the length of a `string`, in terms of number of characters.
+
+## Syntax
+
+```pseudo
+int length = str.length();
+```
+
+Here, `str` is a string variable or a string literal, and length is an integer variable that will hold the length of the string.
+
+## Example
```

Comment:
We'd like a sentence of description preceding the example describing what it's doing.